src/SamplesApp/UITests.Shared/Windows_UI_Xaml/xBindTests/xBind.xaml.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Uno.UI.Samples.Controls;
+using Windows.UI.Xaml.Controls;
+
+namespace UITests.Shared.Windows_UI_Xaml.xBind
+{
+	[SampleControlInfo("XAML", "xBind")]
+	public sealed partial class xBind : UserControl
+	{
+		public XbindViewModel ViewModel { get; set; }
```

Comment:
you can pass the viewmodel type in the SampleControlInfo, instead of doing it manually.
i.e. `[SampleControlInfo("Button", "Hyperlink_Button", typeof(ButtonTestsViewModel))]`

Reply:
xBind won't work without the ViewModel declared in the code-behind, just tried it to make sure.

Reply:
makes sense - `x:Bind` needs to be resolvable at compile time.